src/sonic_ax_impl/mibs/vendor/cisco/ciscoEntityFruControlMIB.py

```diff
@@ -62,27 +115,23 @@ def get_next(self, sub_id):
         :param sub_id: The 1-based snmp sub-identifier query.
         :return: the next sub id.
         """
-        if not self.psuutil:
-            return None
-
         if not sub_id:
             return (1,)
 
-        psu_index = self._getPsuIndex(sub_id)
+        psu_index = self._get_psu_index(sub_id)
```

Comment:
>_get_psu_index [](start = 25, length = 14)

`get_next` is nothrow function. However, if the StateDB is not as good as you expect, there will be exception.

This is a sample:
```
  File "/usr/local/lib/python3.6/dist-packages/sonic_ax_impl/mibs/vendor/cisco/ciscoEntityFruControlMIB.py", line 102, in _get_psu_index
    num_psus = self._get_num_psus()
  File "/usr/local/lib/python3.6/dist-packages/sonic_ax_impl/mibs/vendor/cisco/ciscoEntityFruControlMIB.py", line 65, in _get_num_psus
    num_psus = get_chassis_data(chassis_info)
  File "/usr/local/lib/python3.6/dist-packages/sonic_ax_impl/mibs/vendor/cisco/ciscoEntityFruControlMIB.py", line 36, in get_chassis_data
    return tuple(chassis_info.get(chassis_field.value, b"").decode() for chassis_field in CHASSISInfoDB)
  File "/usr/local/lib/python3.6/dist-packages/sonic_ax_impl/mibs/vendor/cisco/ciscoEntityFruControlMIB.py", line 36, in <genexpr>
    return tuple(chassis_info.get(chassis_field.value, b"").decode() for chassis_field in CHASSISInfoDB)
AttributeError: 'NoneType' object has no attribute 'get'
```